app/gui2/shared/ast/tree.ts

```diff
@@ -1428,7 +1428,7 @@ export class MutableTextLiteral extends TextLiteral implements MutableAst {
   setRawTextContent(rawText: string) {
     let boundary = this.boundaryTokenCode()
     const isInterpolated = this.isInterpolated()
-    const mustBecomeInterpolated = !isInterpolated && (!boundary || rawText.includes(boundary))
+    const mustBecomeInterpolated = !isInterpolated && (!boundary || rawText.match(/["\n\r]/))
```

Comment:
I suppose we will eventually want to transform those multiline strings to `"""` blocks instead, but this does work for now.